templates/ra_rdgw_autoscale_public_lb.template.cfn.json

```diff
@@ -790,6 +851,10 @@
                                 "command": "powershell.exe \"Install-WindowsFeature RDS-Gateway,RSAT-RDS-Gateway -Verbose\"",
                                 "waitAfterCompletion": "0"
                             },
+                            "15-restart-cwa": {
```

Comment:
This isn't needed in the rdcb and rdsh templates, because they reboot the machine after the cloudwatch agent is installed